public/coda/coding_progress.js

```diff
@@ -19,7 +19,37 @@ function update_progress_ui(data) {
         rw.insertCell().innerText = dataset_id
         rw.insertCell().innerText = messages_count
         rw.insertCell().innerText = messages_with_label
-        rw.insertCell().innerText = (100 * messages_with_label / messages_count).toFixed(2) + "%"  
+        rw.insertCell().innerText = (100 * messages_with_label / messages_count).toFixed(2) + '%'
         console.log(dataset_id, messages_count, messages_with_label);
-      }
+    //Formating the rows based on cell value
+        $(document).ready(function(){
+            //Grab the cells of the last rows
+            $('#codingtable td:nth-child(4)').each(function() {
+               var Done = $(this).text();
+                //Style the entire row conditionally based on the cell value
+                if ((parseFloat(Done) >= 0) && (parseFloat(Done) <= 25)) {
```

Comment:
Could we add another code for 0? The difference between 'not started' and 5% complete is pretty significant and I'd like to be able to see it at a glance from the table. Maybe a very pale red would be good for 0.

Reply:
Have added a new class for  0 % 'not started'
![image](https://user-images.githubusercontent.com/23135322/49287493-b8665080-f4ae-11e8-97d1-f0bae87d9dcd.png)